File: FMAplication/ClientApp/src/app/Shared/Entity/DailyTasks/dailySurveyTask .ts
import { SurveyQuestionSet } from "..";
import { DailyBaseTask } from "./dailyBaseTask";
import { DailySurveyTaskAnswer } from "./dailySurveyTaskAnswer";

export class DailySurveyTask extends DailyBaseTask{
    surveyQuestionSet: SurveyQuestionSet;
    surveyQuestionSetId: number;
    dailySurveyTaskAnswers: DailySurveyTaskAnswer[];
}

File: FMAplication/Domain/DailyTasks/DailySurveyTask.cs
using System;
using System.Collections.Generic;
using System.Security.AccessControl;
using FMAplication.Core;
using FMAplication.Domain.Bases;
using FMAplication.Domain.Questions;
using FMAplication.Domain.Sales;
using FMAplication.Enumerations;

namespace FMAplication.Domain.DailyTasks
{
    public class DailySurveyTask : DailyBaseTask
    {
        public SurveyQuestionSet SurveyQuestionSet { get; set; }
        public int SurveyQuestionSetId { get; set; }
        public List<DailySurveyTaskAnswer> DailySurveyTaskAnswers { get; set; }

    }
}